dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/imports/bundle/persister/DefaultTrackerObjectsDeletionService.java

```diff
@@ -96,10 +93,12 @@ public TrackerTypeReport deleteEnrollments(List<UID> enrollments) throws NotFoun
           enrollment.getEvents().stream().filter(event -> !event.isDeleted()).map(UID::of).toList();
       deleteEvents(events);
 
-      RelationshipQueryParams params = RelationshipQueryParams.builder().entity(enrollment).build();
       List<UID> relationships =
-          relationshipStore.getByEnrollment(enrollment, params).stream().map(UID::of).toList();
```

Comment:
Why was this needed before?

Reply:
That was needed to compensate the odd merge strategy present in the deduplication service.
Now we do not need to get the `relationshipItem`s using the service, they are already present in the Hibernate object.